app/deployments/routes.py

```diff
@@ -18,7 +18,7 @@
 import random
 import string
 import uuid as uuid_generator
-from re import search
+from re import match, search
```

Comment:
In general, I would prefer to only `import re` and use `re.match` and `re.search`, especially in such a huge file like this one. `match` and `search` are very wide terms that can be associated with a infinite number of things.
Using `re.match` or `re.search` clearly highlights the fact that we are talking about regex